server/team/team.model.js

```diff
@@ -1,16 +1,23 @@
 const mongoose = require('mongoose');
+
+const userModel = require('../user/user.schema');
 /**
  * Team Schema
  */
+const LighthouseSchema = new mongoose.Schema({
+  seo: Number,
+  'best-practices': Number,
+  accessibility: Number,
+  pwa: Number,
+  performance: Number
+});
+
 const TeamSchema = new mongoose.Schema({
   name: {
     type: String,
     required: true
   },
-  collaborators: {
-    type: [String],
-    required: true
-  },
+  collaborators: [userModel],
```

Comment:
If we want avatars, it will be easiest to store team member data in the team document.

Reply:
We don't need all team members to sign up and join a team, we could just have their usernames and hit a url for their avatar like so:

`https://github.com/tbushman.png` 

![tbushmans avatar](https://github.com/tbushman.png)